### PR TITLE
fix some inconsistencies

### PR DESCRIPTION
**Pull request by sly7-7**
Not sure about all things, so please correct me if I'm wrong.

### CODE REVIEW

File: guides/release/routing/specifying-a-routes-model.md [hunk 1]
```diff
@@ -277,7 +276,7 @@ Routes without dynamic segments will always execute the model hook.
 Sometimes you need to fetch a model, but your route doesn't have the parameters, because it's
 a child route and the route directly above or a few levels above has the parameters that your route
 needs.
-You might run into this if you have a URL like `/photos/4/comments/18`, and when you're in the comments route, you need a photo ID.
+You might run into this if you have a URL like `/album/4/songs/18`, and when you're in the comments route, you need a photo ID.
```
Comment (jenweber) [inline guides/release/routing/specifying-a-routes-model.md]:
```suggestion
You might run into this if you have a URL like `/album/4/songs/18`, and when you're in the songs route, you need an album ID.
```

**Comment by jenweber** [comment]
There is one change that needs to be made before we can merge, in the code suggestions above. If you would like maintainers to be able to fix typos and things in order to get your PR merged faster, click the checkbox "allow maintainers to make edits" when you open a PR. It's not required but it helps! I'm not able to apply this suggestion myself with the current settings.

**Comment by sly7-7** [comment]
@jenweber Sorry for missing that. I wish this feature be enabled by default... Thank you for the review, this should be good now.

**Comment by sly7-7** [comment]
> We haven't really tried to enforce any consistency with `;` or `"` but somehow we should try to lint for this in the future.

For that, I guess that's my vscode config, or the project editor config. VSCode has done the transform on save.

**Pull request merged**